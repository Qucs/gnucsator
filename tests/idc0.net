Idc:I1 gnd _net1 I="8 mA"
R:R5 gnd _net1 R="200"
.TR:TR1 Type="lin" Start="0" Stop="10 us" Points="200" IntegrationMethod="Trapezoidal" Order="2" InitialStep="1 ns" MinStep="1e-16" MaxIter="150" reltol="0.001" abstol="1 pA" vntol="1 uV" Temp="26.85" LTEreltol="1e-3" LTEabstol="1e-6" LTEfactor="1" Solver="CroutLU" relaxTSR="no" initialDC="yes" MaxStep="0"
