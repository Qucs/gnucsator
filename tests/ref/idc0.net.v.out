
Idc #(.I(8m)) I1 (.1(0),.2(_net1));
R #(.R(200)) R5 (.p(0),.n(_net1));
